Integrations/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM deephaven/java-and-python:local-build as install-wheels
+
+RUN set -eux; \
+    apt-get -qq update; \
+    apt-get -qq -y --no-install-recommends install liblzo2-2 curl; \
```

Comment:
consider versioning these, so the dockerfile command is different to handle updates rather than just take it for granted that the docker build cache is empty.

Reply:
Versioning the package installs? ie, `liblzo2-2=x.y.z curl=...`?

Reply:
Yes, otherwise when apt-get installs something new, we won't detect it and update local cached images. It isn't likely to break things... except when it does, and we won't have visibility, will have to either push a garbage step above (or in) the RUN, or tell downstream users to clear the docker caches.

Reply:
There's a scale between being very specific to maximize "reproducibility" vs having generically valid code that should be expected to work when upstream packages have changes. I'm not too keen on trying to be shepherds of package manager versions. It's somewhat nice to have "loose" dependencies that automatically update to their latest versions without needing a lot of engineering management around them.

It's true that you'll need to explicitly bust the cache to pick up changes, or you'll implicitly pick up changes when something earlier causes a cache bust; and when things do break it will take some image inspection to list out all of the differences in system packages.

We *could* potentially think about doing a single `apt-get update` very high up in the stack and not deleting it - with the expectation we explicitly bust that part of the cache after a new major/minor release - such that any downstream installs should always install the same.

That said, I don't think I've seen these patterns in use elsewhere (neither being very specific with 3rd party packages, nor the persisted apt-get update). Not that I'm not willing to try if we really think it's valuable.

Related: `apt-get upgrade` and `apt-get dist-upgrade` have larger implications wrt reproducibility as well.